### PR TITLE
feat: update bug report

### PR DESCRIPTION
**Pull request by mark-kascheev**
## Changes
Added report template for clear bug reporting

Depends on #45 

## Checklist for self-check
- [x] Commits and PRs have been filed according to [the rules on the project](https://github.com/surfstudio/surf-flutter-app-template#workflow-in-a-repository).
- [x] The author is marked as an assigne and assigned mandatory reviewers.
- [x] Required labels marked
- [ ] Specified related tasks and/or related PRs.
- [x] Specified Changes.
- [ ] Attached videos/screenshots demonstrating the fix/feature.
- [x] All unspecified fields in the PR description deleted.
- [ ] New code covered by tests.

## Checklist for reviewers
- [ ] CI passed successfully _(with a green check mark)_.
- [ ] PR is atomic, by volume no more than 400 (+-) corrected lines (not including codogen).

Design:
- [ ] System design corresponds to the agreements on structure and architecture on the project.
- [ ] The code is decomposed into necessary and sufficient components.

Functionality:
- [ ] The code solves the problem.
- [ ] Any changes to the user interface are reasonable and look good.

Complexity:
- [ ] The code is clear, easy to read, functions are small, no more than 50 lines.
- [ ] The logic is not overcomplicated, there is no overengineering (no code sections that may be needed in the future, but no one knows about it).

Tests:
- [ ] Updated or added tests for mandatory components.
- [ ] The tests are correct, helpful, and well designed/developed.

Naming:
- [ ] The naming of variables, methods, classes and other components is understandable.

Comments:
- [ ] The comments are understandable and helpful.

Documentation:
- [ ] All labels are correct
- [ ] Technical documentation updated (after approval, updates last reviewer).

### CODE REVIEW

File: .fvm/flutter_sdk [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/parshonok/fvm/versions/3.7.6
```
Comment (dshevchenkoo) [inline .fvm/flutter_sdk:1]: add to ignore

Comment (mark-kascheev) [reply .fvm/flutter_sdk:1]: I fixed it in SNP-271

**Comment by plasticfiresam** [comment]
WIP until we make a decision about Bug report and PR templates

**Comment by plasticfiresam** [comment]
Closed, we will implement template on all of our plugins in one time

**Pull request closed without merging**